packages/catalog-process/test/integration/updateEServiceSignalhubFlag.test.ts

```diff
@@ -0,0 +1,191 @@
+/* eslint-disable @typescript-eslint/no-floating-promises */
+import {
+  decodeProtobufPayload,
+  getMockContext,
+  getMockAuthData,
+  getMockDocument,
+  getMockDescriptor,
+  getMockEService,
+  getMockDelegation,
+} from "pagopa-interop-commons-test";
+import {
+  Descriptor,
+  descriptorState,
+  EService,
+  toEServiceV2,
+  EServiceSignalHubEnabledV2,
+  EServiceSignalHubDisabledV2,
+  operationForbidden,
+  delegationState,
+  delegationKind,
+} from "pagopa-interop-models";
+import { expect, describe, it } from "vitest";
+import {
+  addOneDelegation,
+  addOneEService,
+  catalogService,
+  readLastEserviceEvent,
+} from "../integrationUtils.js";
+import { eServiceNotFound } from "../../src/model/domain/errors.js";
+
+describe("update E-service signalhub flag for an already created E-service", async () => {
```

Comment:
I would also add tests for the `false` >> `false` and `true` >> `true` cases, and in these cases the method should probably succeed but not create any event